src/ansys/additive/core/simulation.py

```diff
@@ -53,6 +53,8 @@ class SimulationStatus(str, Enum):
     """Simulation completed with warnings."""
     ERROR = "Error"
     """Simulation errored before completion."""
```

Comment:
I would prefer "Simulation encountered an error before completing." (But not a big deal.)